### ISSUE TITLE
PID is not respected on MacOS/Darwin platform

### ISSUE THREAD

**Issue by edariedl** [schneems/get_process_mem#39]
get_process_mem always returns memory used by current process. It does not respect PID.

Here are some tests from `irb`
```
irb(main):001:0> require 'get_process_mem'
=> true
irb(main):002:0> GetProcessMem.new.kb
=> 17744.0
irb(main):003:0> `ps -o rss= -p #{Process.pid}`
=> " 17836\n"
```
but if I try to get memory of different process
```
irb(main):004:0> GetProcessMem.new(84330).kb
=> 17860.0
irb(main):005:0> `ps -o rss= -p 84330`
=> " 75104\n"
```
I'll get back the same amount of memory as I've got for the current process.

This issue was introduced by the pull request https://github.com/schneems/get_process_mem/pull/32. Code for Darvin platform doesn't use PID to determine used RAM, it always returns used RAM for current process.

**Comment by schneems** [comment]
I've got a workaround on https://github.com/schneems/get_process_mem/pull/41

**Comment by schneems** [comment]
This is fixed and released in 0.2.6
